### PR TITLE
Typo/grammatical changes.

### PR DESCRIPTION
**Pull request by PoojaChandak**

### CODE REVIEW

File: doc/apimanager-reference.md
Comment (miguelsorianod) [inline doc/apimanager-reference.md:281]: I think this should actually be:
`the following secrets have to be pre-created by the user:`

**Review by miguelsorianod — CHANGES_REQUESTED** [review]
Hi @PoojaChandak,

Thanks for you PR.

I've left a minor change request. Overall looks good to me :+1:

File: doc/apimanager-reference.md
Comment (PoojaChandak) [reply doc/apimanager-reference.md]: Change incorporated. Thanks.

**Comment by PoojaChandak** [comment]
@miguelsorianod Made the suggested changes. Please check. Thanks.

File: doc/apimanager-reference.md
Comment (miguelsorianod) [inline doc/apimanager-reference.md:281]: Double ':' typo?

Comment (PoojaChandak) [reply doc/apimanager-reference.md]: Oops! Corrected.
Thanks.

Comment (miguelsorianod) [reply doc/apimanager-reference.md]: :+1:

**Review by miguelsorianod — CHANGES_REQUESTED** [review]
Contents look good to me now :+1: 

Could you squash all the commits in a single commit?

Thank you.

**Comment by PoojaChandak** [comment]
@miguelsorianod Please check now.

**Comment by miguelsorianod** [comment]
Hi @PoojaChandak,

It seems commits have not been squashed. I see 4 commits :thinking: 

This guide might help you: https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History

Specifically, section "Changing Multiple Commit Messages", subsection "Squashing Commits"

**Comment by PoojaChandak** [comment]
looks like now it is correctly done. Please confirm. :)

**Review by miguelsorianod — APPROVED** [review]
Looks good now :+1: 

Let's wait until CI finishes and then I'll merge it

**Comment by PoojaChandak** [comment]
Thanks for the help.  @miguelsorianod

**Pull request merged**
